0_portfolio_input_check_functions.R

```diff
@@ -1531,7 +1531,7 @@ calculate_portfolio_emissions <- function(
 add_other_to_sector_classifications <- function(audit) {
   # fix sector classifications
   audit <- audit %>%
-    mutate(ald_sector = ifelse(mapped_sector != "Other" & is.na(ald_sector), mapped_sector, ald_sector))
+    mutate(ald_sector = ifelse(financial_sector != "Other" & is.na(ald_sector), financial_sector, ald_sector))
```

Comment:
I don't remember why but I think `dplyr::if_else()` has some advantage over `ifelse()`.

Reply:
if_else expects that the types are going to be the same ie it's replacing a character with a character so it throws more errors, which probably isn't a bad thing, but I sometimes ignore this. I think there are issues also with replacing NAs with characters, but I'm not sure.

Reply:
yes, @Clare2D is exactly right... `dplyr::if_else()` is "safer" because it type checks everything... which can lead to more errors (which some may say is a good thing), and which may require you to return type specific `NA`s... like `NA_character_` instead of just `NA`